ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -21,7 +21,23 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-ci-tests-kube-burner
+- as: ocp-qe-perfscale-aws-ci-tests-medium-scale-kube-burner
+  cron: 0 12 * * 2
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      SIZE_VARIANT: large
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: m5.2xlarge
```

Comment:
I know we are porting from airflow, but m5.xlarge are enough for this test.

Reply:
hey @afcollins can you send me the results from said tests showing this?

Reply:
Updated @afcollins based on your recommendation! Thanks!!